client/stylesheets/_toCat.scss

```diff
@@ -3272,6 +3272,19 @@ On click the images will be displayed at normal size to complete the effect
     height: 500px !important;
 }
 
+.itemAfterBanner {
+    background-color: $color-purple;
+    padding: 10px;
+    width: 100%;
+    border-radius: 4px;
+    margin-bottom: 25px;
+}
+
+.itemAfterBanner b {
```

Comment:
Merge rule `.itemAfterBanner b` with rule on line 3275<br>Selector `itemAfterBanner` should be written in lowercase with hyphens